components/textbox/index.ts

```diff
@@ -30,12 +32,20 @@ const makeMapStateToProps = () => {
 
     return (state: GlobalState, ownProps: Props) => {
         const teamId = getCurrentTeamId(state);
+        const license = getLicense(state);
+        const useGroupMentions = license?.IsLicensed === 'true' && license?.LDAPGroups === 'true' && haveIChannelPermission(state, {
+            channel: ownProps.channelId,
+            team: teamId,
+            permission: Permissions.USE_GROUP_MENTIONS,
+        });
+        const autocompleteGroups = useGroupMentions ? getAssociatedGroupsForReference(state, teamId, ownProps.channelId) : [];
```

Comment:
Same here. You can also make a selector to return a memoized copy of [] instead

Reply:
Updated.